cmd/minishift/cmd/start.go

```diff
@@ -410,9 +411,14 @@ func exportContainerImages(hostVm *host.Host, version string) {
 }
 
 // calculateDiskSizeInMB converts a human specified disk size like "1000MB" or "1GB" and converts it into Megabits
-func calculateDiskSizeInMB(humanReadableDiskSize string) int {
-	diskSize, err := units.FromHumanSize(humanReadableDiskSize)
+func calculateDiskSizeInMB(humanReadableSize string) int {
+	if stringUtils.HasOnlyNumbers(humanReadableSize) {
+		humanReadableSize += "MB"
+	}
+
+	diskSize, err := units.FromHumanSize(humanReadableSize)
 	if err != nil {
+		fmt.Println("")
```

Comment:
@gbraad We can have only `fmt.Println()` as part of new line if that only we want.